bokeh/application/application.py

```diff
@@ -100,9 +101,23 @@ class Application(object):
     # will function without bringing in tornado.
     _is_a_bokeh_application_class = True
 
-    def __init__(self, *handlers):
+    def __init__(self, *handlers, metadata=None):
```

Comment:
Are you OK if I go:
 def __init__(self, *handlers, **kwds):
       metadata = kwds.get('metadata', None)

?